docs/governance/cardano-governance/submitting-governance-actions.md

```diff
@@ -0,0 +1,254 @@
+  - **Title:** A concise title for the governance action.
+  - **Abstract:** A brief summary of the proposal.
+  - **Motivation:** A concise statement of the problem or opportunity that the governance action addresses.
+  - **Rationale:** A logical argument explaining why the proposed solution is the best approach to address the stated motivation.
+  - **References:** Links to any relevant documents, research, or discussions, ideally all references should also be hosted on a content-addressable storage solution like [IPFS](https://ipfs.tech/).
+- **Be hosted on Immutable Storage:** Use a content-addressable storage solution like [IPFS](https://ipfs.tech/). This ensures the content cannot be altered after submission. See [Hosting Your Metadata on Immutable Storage](#immutable-metadata-hosting) below.
+
+:::important
+For practical examples of governance action metadata, see the [**Governance Action Metadata Examples**](#governance-action-metadata-examples) section below.
+:::
+
+---
+
+## Immutable Metadata Hosting
+
+To ensure transparency and permanence, your metadata file and references should remain immutable. Here’s how to do this, starting with your metadata hosted in a GitHub repository:
+
+### 1. Host Your Metadata File in a GitHub Repository
+
+- **Set Up a GitHub Repository:**  
+  Create a new repository using GitHub’s guide: [Create a New Repository on GitHub](https://docs.github.com/en/get-started/quickstart/create-a-repo)
+- **Upload and Commit Your Metadata File:**  
+  Add your metadata file (e.g., JSON or JSON-LD) to your repository and commit it. Each commit generates a unique hash that permanently identifies that version of your file. [Git Commit Guide](https://github.com/git-guides/git-commit)
+
+### 2. Obtain a Permanent GitHub Link
+
+- **Generate an Immutable Link:**  
+  Navigate to your metadata file in your repository and click the **"Raw"** button. Copy the URL from your browser’s address bar, then replace the branch name with the commit hash. Detailed instructions can be found here: [Getting Permanent Links to Files on GitHub](https://docs.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files)
+
+### 3. Pin Your Metadata to IPFS
+
+Pinning your metadata to IPFS ensures that it is stored on a decentralized network and remains available permanently:
+
+- **Using the NMKR API:**  
+  Follow the NMKR API instructions for pinning files to IPFS. [NMKR API Tutorial for Pinning Files to IPFS](https://docs.nmkr.io/nmkr-studio-api/api-features#ipfs)
+- **Using the Blockfrost Tutorial:**  
+  Use Blockfrost’s guide to pin your metadata via their API. [Blockfrost Tutorial for Pinning Files to IPFS](https://blockfrost.dev/start-building/ipfs/)
+- **Using IPFS:**  
+  Follow the official IPFS guide to pin your files manually. [IPFS Pinning Guide](https://docs.ipfs.io/how-to/pin-files/)
+
```

Comment:
```suggestion

To ensure fast and reliable access, it is recommended to use multiple different pinning services.
```
This is something we do for Intersect actions, might be nice to include if you agree?